health-services/project-factory/src/server/utils/onGoingCampaignUpdateUtils.ts

```
@@ -0,0 +1,806 @@
+
+function unhideColumnsOfProcessedFile(sheet: any, columnsToUnide: any) {
+  columnsToUnide.forEach((column: any) => {
+    if (column) {
+      sheet.getColumn(column).hidden = false;
+    }
+  });
+}
+
+function modifyNewSheetData(processedDistrictSheetData: any, newSheetData: any, headers: any, oldTargetColumnsToHide: any[], localizationMap?: any) {
+  let modifiedData = [];
+  let localizedHeaders = getLocalizedHeaders(headers, localizationMap);
```

Biome

[error] This let declares a variable that is only assigned once.

'localizedHeaders' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)